packages/protocol/test/common/multisig.ts

```diff
@@ -167,17 +258,29 @@ contract('MultiSig', (accounts: any) => {
     it('should not allow an external account to replace an owner', async () => {
       await assertRevert(multiSig.replaceOwner(accounts[1], accounts[2]))
     })
+
+    it('should not allow an owner to be replaced by the null address', async () => {
```

Comment:
Ditto, this is failing because it's not being called by the multisig. We're not actually testing what we intend to test here

Reply:
redesigned